src/modules/item/utils.ts

```diff
@@ -158,3 +145,29 @@ export async function generateImage(item: Item, width = 1024, height = 1024) {
   canvas.toBlob(result => (result ? blob.resolve(result) : blob.reject(new Error('Error generating image blob'))))
   return blob
 }
+
+export function isComplete(item: Item) {
+  return !isEditable(item) && !!item.beneficiary && !!item.price
+}
+
+export function isEditable(item: Item) {
+  const data = item.data as WearableData
+  return !item.rarity || !data.category
+}
+
+export function isOwner(item: Item, address?: string) {
```

Comment:
There's a lib file named address that has a few utils for these address comparisons. You can use isEqual from there!